jax/_src/api.py

```diff
@@ -137,9 +137,11 @@ def _update_debug_special_thread_local(_):
 
 float0 = dtypes.float0
 
+_P = ParamSpec("_P")
```

Comment:
reuse this from stages instead of redefining in 3 files.

Reply:
Thanks! Addressed in 32de804c5c320942ef4b93d1f18751391edc2b27 (rebased to keep everything in a single commit)